packages/json-schema/test/discriminator.test.ts

```diff
@@ -0,0 +1,355 @@
+import { deepStrictEqual, ok, strictEqual } from "assert";
+import { describe, it } from "vitest";
+import { emitSchema } from "./utils.js";
+
+describe("json-schema: discriminator", () => {
```

Comment:
you can remove this top level describe, we used to have that before when mocha would merge all the test together but now its already scopped per file